gslib/tests/test_cp.py

```diff
@@ -495,20 +495,6 @@ def test_noclobber_different_size(self):
       self.assertIn('Skipping existing item: %s' % suri(f), stderr)
       self.assertEqual(f.read(), b'quux')
 
-  @SequentialAndParallelTransfer
```

Comment:
My test! Don't kill it! ;)

Reply:
Oh!! Sorry, I thought I'd messed up the merge when we saw there was two of those! Thanks for letting us know that's intentionally there lol :smiley:

Reply:
@houglum - there are intentionally 2 tests called test_noclobber_different_size() ?

Reply:
Derp. I was looking at the master branch (where this wasn't duplicated) and wondering why you removed it -- looks like this test actually got added in twice in the py-six-current branch.  Thanks for catching that :)